packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -588,6 +588,8 @@ export const messages = {
   package_id_gatherer_placeholder: '04t...',
   force_function_enter_function: 'Enter a name for the function',
   force_function_enter_language: 'Select a language for your function',
-  force_function_pull_dependencies_error:
+  force_function_install_npm_dependencies_progress:
+    'Installing NPM dependencies',
```

Comment:
@ekapner This is a usability improvement to show a progress indicator here when installing dependencies after running "SFDX: Create Function"